src/main/java/core/basesyntax/service/impl/WriterFileImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriterFile;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriterFileImpl implements WriterFile {
+    private static final String OUTPUT_FILE_NAME = "src/main/resources/report.csv";
```

Comment:
Please make method argument in order to have flexibility in saving to different files